### PR TITLE
Bugfix/code smells

### PR DESCRIPTION
**Pull request by p-ranav**

### CODE REVIEW

File: apps/wear/smartdrive/app/services/bluetooth.service.ts [hunk 1]
```diff
@@ -250,9 +250,6 @@ export class BluetoothService {
       name: argdata.name
     };
     Log.D(`${peripheral.name}::${peripheral.address} - discovered`);
-    if (this.isSmartDrive(peripheral)) {
-      const sd = this.getOrMakeSmartDrive(peripheral);
-    }
```
Comment (finger563) [inline apps/wear/smartdrive/app/services/bluetooth.service.ts]: This should probably be added back - the function `getOrMakeSmartDrive(...)` modifies a static `SmartDrives` array - so this function actually changes the system state (and should) - we should probably just more clearly not use the return value.

File: apps/mobile/smart-eval/src/services/bluetooth.service.ts [hunk 2]
```diff
@@ -465,9 +465,6 @@ export class BluetoothService {
       address: argdata.UUID,
       name: argdata.name
     };
-    if (this.isSmartDrive(peripheral)) {
-      const sd = this.getOrMakeSmartDrive(peripheral);
-    }
```
Comment (finger563) [inline apps/mobile/smart-eval/src/services/bluetooth.service.ts]: This should probably be added back - the function `getOrMakeSmartDrive(...)` modifies a static `SmartDrives` array - so this function actually changes the system state (and should) - we should probably just more clearly not use the return value.

File: apps/wear/smartdrive/app/services/bluetooth.service.ts (hunk 1 above)
Comment (p-ranav) [reply apps/wear/smartdrive/app/services/bluetooth.service.ts]: Okay, I'll just remove `const sd =`

File: apps/mobile/smart-eval/src/services/bluetooth.service.ts (hunk 2 above)
Comment (p-ranav) [reply apps/mobile/smart-eval/src/services/bluetooth.service.ts]: Okay, I'll just remove `const sd =`

File: apps/wear/pushtracker/app/models/activity-detector.ts [hunk 3]
```diff
@@ -237,8 +237,6 @@ export class ActivityDetector {
         maxZ = z;
       }
     });
-    const jerk = maxZ - minZ;
```
Comment (finger563) [inline apps/wear/pushtracker/app/models/activity-detector.ts]: If we're removing `jerk`, then the `maxZ` and `minZ` are likely also unused and can be removed.

Comment (p-ranav) [reply apps/wear/pushtracker/app/models/activity-detector.ts]: Looks like it. Removing shortly.

**Pull request merged**